click/_compat.py

```diff
@@ -7,8 +7,9 @@
 
 
 PY2 = sys.version_info[0] == 2
-WIN = sys.platform.startswith('win')
 CYGWIN = sys.platform.startswith('cygwin')
+APP_ENGINE = os.environ.get('SERVER_SOFTWARE','').startswith('Development')
```

Comment:
space after `,`

Reply:
Also, what's the full name of the App Engine `SERVER_SOFTWARE`? Starting with "Development" sounds extremely generic/vague.

Reply:
Can definitely fix the stylistic issue. We'll determine what the proper way of establishing a development version of GAE is. According to Google's own docs, they do something similar to determine whether we're running in a production environment (from: https://cloud.google.com/appengine/docs/standard/python/tools/using-local-server):

```
if os.getenv('SERVER_SOFTWARE', '').startswith('Google App Engine/'):
  # Production
else:
  # Local development server
```